app/javascript/gobierto_TEMP/webapp/Maker.vue

```diff
@@ -0,0 +1,198 @@
+          for="dashboard-status"
+        >
+          <input
+            id="dashboard-status"
+            type="checkbox"
+            @input="handleInputStatus"
+          >
+          {{ publicLabel }}
+        </label>
+      </template>
+    </HeaderForm>
+
+    <Main>
+      <template #aside>
+        <Aside>
+          <SmallCard
+            v-for="({ name }, key) in cards"
+            :key="key"
+            :name="name"
+            @drag.native="drag(key, $event)"
+            @dragend.native="dragend"
+          />
+        </Aside>
+      </template>
+
+      <Viewer
+        v-if="configuration"
+        ref="viewer"
+        :is-draggable="true"
+        :is-resizable="true"
+        :item="item"
+        :config="configuration"
+        @layout-updated="handleViewerUpdated"
+        @layout-ready="handleViewerReady"
+      />
+    </Main>
+  </div>
+</template>
+
+<script>
+// add the styles here, because this element can be inserted both as a component or standalone
+import "../../../assets/stylesheets/module-TEMP-maker.scss";
+import Viewer from "./Viewer";
+import HeaderForm from "./components/HeaderForm";
+import Aside from "./layouts/Aside";
+import Main from "./layouts/Main";
+import SmallCard from "./components/SmallCard";
+import { Widgets } from "./lib/widgets";
+import { DashboardFactoryMixin } from "./lib/factories";
+import { TextEditable } from "lib/vue-components";
+
+export default {
+  name: "Maker",
+  components: {
+    Viewer,
+    HeaderForm,
+    Main,
+    Aside,
+    SmallCard,
+    TextEditable,
+  },
+  mixins: [DashboardFactoryMixin],
+  data() {
+    return {
+      dirty: false,
+      cards: Widgets,
+      item: null,
+      configuration: null,
+      viewerLoaded: false,
+      publicLabel: "Público",
+    };
+  },
+  computed: {
+    id() {
+      return this.$root.$data?.id;
+    },
+    title() {
+      return this.configuration?.data?.attributes?.title || "Título del dashboard";
+    },
+    status() {
+      return this.configuration?.data?.attributes?.visibility_level === "active";
+    },
+  },
+  created() {
+    this.getConfiguration();
+  },
+  mounted() {
+    document.addEventListener('dragover', this.dragoverPosition)
+  },
+  destroyed() {
+    document.removeEventListener('dragover', this.dragoverPosition)
+  },
+  methods: {
+    async getConfiguration() {
+      this.configuration = this.id ? await this.getDashboard(this.id) : {};
+      this.dirty = false
+    },
+    dragoverPosition({ clientX, clientY }) {
+      this.x = clientX
+      this.y = clientY
+    },
+    drag(key) {
+      if (!this.item) {
+        const match = Widgets[key];
+        if (match) {
+          this.item = {
+            i: `${key}-${Math.random().toString(36).substring(7)}`,
+            ...match
+          };
+        }
+      } else {
+//         const ix = this.configuration?.data?.attributes?.widget_configuration?.findIndex(({ i }) => i === this.item.i)
```

Comment:
Do you need it in the future?

Reply:
Yep, I pointed in a new issue https://github.com/PopulateTools/gobierto/issues/3533